cmd/palomar/README.md

```diff
@@ -1,37 +1,33 @@
-# palomo
+# Palomar
 
-An elasticsearch/opensearch frontend and ATP repo crawler meant to provide
-search services for the bluesky network.
+Palomar is an Elasticsearch/OpenSearch frontend and ATP (ActivityPub) repository crawler designed to provide search services for the Bluesky network.
+
+## Prerequisites
+
+- GoLang (version 1.x)
+- Running instance of Elasticsearch or OpenSearch for indexing.
+- Valid credentials for the Personal Data Store (PDS) you want to index against.
```

Comment:
you don't need PDS credentials. this service consumes from a full firehose feed, which doesn't not require authentication.